src/autobuild-action.ts

```diff
@@ -62,16 +72,35 @@ async function run() {
       return;
     }
 
-    const config = await config_utils.getConfig(
-      getTemporaryDirectory(),
+    const apiDetails = {
+      auth: getRequiredInput("token"),
+      externalRepoAuth: getOptionalInput("external-repository-token"),
+      url: getRequiredEnvParam("GITHUB_SERVER_URL"),
+      apiURL: getRequiredEnvParam("GITHUB_API_URL"),
+    };
```

Comment:
I think this would be simpler. `externalRepoAuth` isn't necessary here.

```suggestion
    const apiDetails = apiClient.getApiDetails();
```

Reply:
We don't have an `apiClient` here I don't think, but we can certainly drop the `externalRepoAuth`.

Reply:
Right...minor point. You can make `getApiDetails` exported in `apiClient.ts`. Not a big deal either way.

Reply:
Oh, I see what you mean now.  Let's do that.